.github/workflows/dev-cluster-check.yml

```diff
@@ -81,6 +81,13 @@ jobs:
         with:
           maven-command: clean verify -Pcucumber -Dparallel -Dgroups=${{ github.event.inputs.groups || ''}} -Devent-bridge.manager.url=$DEV_CLUSTER_MANAGER_URL -Dkeycloak.realm.url=https://sso.redhat.com/auth/realms/redhat-external -Dbridge.client.id=$DEV_SSO_CLIENT_ID -Dbridge.client.secret=$DEV_SSO_CLIENT_SECRET -Dmanaged.kafka.sso.auth-server-url=https://sso.redhat.com/auth/realms/redhat-external -Dtest.credentials.file=e2e-test-config.yaml -Dfail.cleanup.disable=${{ github.event.inputs.fail-cleanup-disable || 'false'}} -Dcleanup.disable=${{ github.event.inputs.cleanup-disable || 'false'}}
           working-directory: integration-tests
+      - name: Archive junit test results
+        uses: actions/upload-artifact@v3
+        with:
+          name: junit-test-results
+          path: |
+            integration-tests/target/failsafe-reports/TEST-*.xml
```

Comment:
just to double check. We only want to store integration test results and not unit tests results?

Reply:
As far as I know we run only the integration tests in our cluster checks. Unit tests are run as part of Bridge - CI workflow.

Reply:
ok, thanks!